Dockerfile

```diff
@@ -13,8 +13,8 @@
 
 FROM gcr.io/google-appengine/debian9:latest
 
-ENV PACKER_VERSION 1.0.0
-ENV PACKER_SHA256 ed697ace39f8bb7bf6ccd78e21b2075f53c0f23cdfb5276c380a053a7b906853
+ENV PACKER_VERSION 1.5.5
```

Comment:
Is this version bumb required by your scenario? If not can you please revert?

Reply:
Yes, it is required.